Sources/FluentBenchmark/Tests/GroupByTests.swift

```diff
@@ -0,0 +1,28 @@
+extension FluentBenchmarker {
+    public func testGroupBy() throws {
+        try self.testGroupBy_join()
+    }
+
+    private func testGroupBy_join() throws {
+        try runTest(#function, [
+            SolarSystem()
+        ]) {
+            let earth = try XCTUnwrap(Planet.query(on: self.database).filter(\.$name == "Earth").first().wait())
+
+            let planets = try Planet.query(on: self.database)
+                .field(\.$id)
+                .join(PlanetTag.self, on: \PlanetTag.$planet.$id == \Planet.$id)
+                .filter(PlanetTag.self, \.$planet.$id == earth.requireID())
+                .all().wait()
+            XCTAssertEqual(planets.count, 2)
+
+            let groupedPlanets = try Planet.query(on: self.database)
```

Comment:
what do you think about throwing in a `count` here? I know we aren't testing aggregation per-se here, but I think the group clause would be less confusing in the context of aggregation. Then maybe name it "tagsPerPlanet" because the count will be the number of tags and the number of rows will be the number of planets.

Reply:
I tried that. The problem is that Fluent does all sorts of weirdness when you run an aggregate query, such as only fetching the `id` field from the table you are querying. `GROUP BY` with an aggregate only really counts when you are fetching more than just the aggregated value.